src/explore-education-statistics-common/src/components/Banner.tsx

```diff
@@ -0,0 +1,47 @@
+import styles from '@common/components/Banner.module.scss';
+import useMounted from '@common/hooks/useMounted';
+import ButtonText from '@common/components/ButtonText';
+import classNames from 'classnames';
+import React, { ReactNode } from 'react';
+
+interface Props {
+  children: ReactNode;
+  showBanner: boolean;
+  wide?: boolean;
+  onClose: () => void;
+}
+
+export default function Banner({
+  children,
+  showBanner,
```

Comment:
What's the benefit of this prop? We can just wrap the banner with a condition to achieve the same thing

Reply:
Yeah good point, I've got rid of it.